### ISSUE TITLE
[BUG] Out of memory error on windows when running the "next" run of the BiMotion experiment

### ISSUE THREAD

**Issue by marcobarilari** [cpp-lln-lab/CPP_BIDS#237]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Expected Behavior

To run the next run without having to close and reopen matlab

### Current Behavior

Matlab complains and I need to close and reopen it

### Error message

Matlab throws this error (apologies for the picture but it is cross computers)
![IMG_9452](https://github.com/cpp-lln-lab/CPP_BIDS/assets/38101692/6a2cdd33-9fc9-45c6-8935-d6e9eab5d2cb)

### Environment

```markdown
- OS: windows 10 pro
- Matlab: 2015b
```

### Branch

```markdown
- branch: (HEAD detached at 627ceef)
- version: v2.1.2dev (from txt file)
```

**Comment by Remi-Gau** [comment]
I think that restarting matlab had worked in the past but I agree this is "suboptimal"

**Comment by marcobarilari** [comment]
Yes it is that one. Now the problem is systematic so for each run I need to close and open Matlab again.

I will try to query which file is problematic next week.
